<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sorologia</class>
 <widget class="QDialog" name="Sorologia">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>359</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>359</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>797</width>
    <height>455</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sorologia</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>641</width>
     <height>301</height>
    </rect>
   </property>
   <property name="title">
    <string>Sorologia</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QGroupBox" name="groupBox_hepatiteB">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>28</y>
      <width>271</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Hepatite B</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="3">
        <widget class="QLineEdit" name="lineEdit_AntiHBs"/>
       </item>
       <item row="0" column="3">
        <widget class="QLineEdit" name="lineEdit_HbsAg"/>
       </item>
       <item row="5" column="3">
        <widget class="QLineEdit" name="lineEdit_AntiHbe"/>
       </item>
       <item row="2" column="3">
        <widget class="QLineEdit" name="lineEdit_AntiHBc"/>
       </item>
       <item row="4" column="3">
        <widget class="QLineEdit" name="lineEdit_HBeAg"/>
       </item>
       <item row="3" column="3">
        <widget class="QLineEdit" name="lineEdit_AntiHBcIgM"/>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_HbsAg">
         <property name="text">
          <string>HbsAg</string>
         </property>
         <property name="buddy">
          <cstring>comboBox_HbsAg</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QComboBox" name="comboBox_AntiHBs">
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Reagente</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Não-Reagente</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_AntiHBc">
         <property name="text">
          <string>Anti-HBc</string>
         </property>
         <property name="buddy">
          <cstring>comboBox_AntiHBc</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QComboBox" name="comboBox_HBeAg">
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Reagente</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Não-Reagente</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QComboBox" name="comboBox_AntiHBc">
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Reagente</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Não-Reagente</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QComboBox" name="comboBox_HbsAg">
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Reagente</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Não-Reagente</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="label_HBeAg">
         <property name="text">
          <string>HBeAg</string>
         </property>
         <property name="buddy">
          <cstring>comboBox_HBeAg</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="label_AntiHBcIgM">
         <property name="text">
          <string>Anti-HBc IgM</string>
         </property>
         <property name="buddy">
          <cstring>comboBox_AntiHBcIgM</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QComboBox" name="comboBox_AntiHbe">
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Reagente</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Não-Reagente</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QComboBox" name="comboBox_AntiHBcIgM">
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Reagente</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Não-Reagente</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="label_AntiHbe">
         <property name="text">
          <string>Anti-Hbe</string>
         </property>
         <property name="buddy">
          <cstring>comboBox_AntiHbe</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_AntiHBs">
         <property name="text">
          <string>Anti-HBs</string>
         </property>
         <property name="buddy">
          <cstring>comboBox_AntiHBs</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkBox_HbsAg">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="checkBox_AntiHBs">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="checkBox_AntiHBc">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="checkBox_AntiHBcIgM">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="checkBox_HBeAg">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QCheckBox" name="checkBox_AntiHbe">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>28</y>
      <width>341</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QGroupBox" name="groupBox_hepetiteC">
       <property name="title">
        <string>Hepatite C</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="2">
           <widget class="QLineEdit" name="lineEdit_AntiHCV"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_AntiHCV">
            <property name="text">
             <string>Anti-HCV</string>
            </property>
            <property name="buddy">
             <cstring>comboBox_AntiHCV</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBox_AntiHCV">
            <item>
             <property name="text">
              <string/>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Reagente</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Não-Reagente</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_HIV">
       <property name="title">
        <string>HIV</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="2">
           <widget class="QLineEdit" name="lineEdit_AntiHIV"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_AntiHIV">
            <property name="text">
             <string>Anti-HIV</string>
            </property>
            <property name="buddy">
             <cstring>comboBox_AntiHIV</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBox_AntiHIV">
            <item>
             <property name="text">
              <string/>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Reagente</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Não-Reagente</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>320</y>
     <width>195</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="pushButton_Confirmar">
      <property name="text">
       <string>Confirmar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Cancelar">
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
